generated/nidigital/nidigital/session.py

```diff
@@ -2694,7 +2694,11 @@ def __exit__(self, exc_type, exc_value, traceback):
     def initiate(self):
         '''initiate
 
-        TBD
+        Starts bursting the pattern, causing the NI-Digital session
+        to be committed. To stop the pattern burst, call abort
+        method. If keep alive pattern is bursting when abort
+        is called, keep alive pattern will not be stopped. To stop
```

Comment:
Change to "If keep alive pattern is bursting when abort is called or upon exiting the context manager, ..."